api/server.spec.js

```diff
@@ -0,0 +1,46 @@
+const request = require('supertest');
+
+const server = require('./server.js');
+
+describe('server.js', () => {
+    describe('GET /api/games', () => {
+        it('Respond with status code 200', async () => {
+            let response = await request(server).get('/api/games');
+            expect(response.status).toBe(200);
+        });
+        it('Respond with a json object', async () => {
+            let response = await request(server).get('/api/games');
+            expect(response.type).toMatch(/json/i);
+        });
+        it('Respond with array of games', async () => {
+            let response = await request(server).get('/api/games');
```

Comment:
The same applies here with making your tests more performant; use a beforeAll and only do the request once!